docs/cluster_spec.md

```diff
@@ -387,6 +387,28 @@ spec:
       image: busybox
 ```
 
+Install cachefiled
```

Comment:
Thanks for this PR!  What is the purpose of adding this documentation?

Reply:
Check this thread https://github.com/kubeflow/experimental-kvc/issues/26

Reply:
Then should we mention that this is for kubeflow?

Reply:
As you want. Generally Is to enable node cache for nfs/efs PVC

Reply:
@bhack actually that is even more valuable data ;)